common/nexus/outgoing_service_registry_test.go

```diff
@@ -345,6 +371,25 @@ func TestCreate_NoURL(t *testing.T) {
 		&operatorservice.CreateNexusOutgoingServiceRequest{
 			Namespace: testNamespace,
 			Name:      testServiceName,
+			Spec: &nexuspb.OutgoingServiceSpec{
+				PublicCallbackUrl: testPublicCallbackURL,
+			},
+		},
+	)
+	require.ErrorContains(t, err, nexus.IssueURLNotSet)
+}
+
+func TestCreate_NoPublicCallbackURL(t *testing.T) {
```

Comment:
I'm ok with it being inconsistent because I doubt we'll make the time to go through and update every test, but this is a nit and I won't be a stickler about it